### PR TITLE
add Hardware Craft

### PR DESCRIPTION
**Pull request by NxrVincent**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/hardwarecraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "server_name": "Hardware Craft",
```
Comment (niklaswa) [inline minecraft_servers/hardwarecraft/manifest.json:2]: Please change the `server_name` to `hardwarecraft` as it must be the same as the directory name.

**Comment by NxrVincent** [comment]
Done

File: minecraft_servers/hardwarecraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "server_name": "hardwarecraft",
+  "nice_name": "Hardware Craft",
+  "direct_ip": "hardware-craft.xyz",
+  "server_wildcards": [
+    "%.hardware-craft.xyz",
+    "%.hardware-network.de"
+  ],
+  "social": {
+    "web": "https://hardware-network.de",
+    "web_support": "https://events.hardware-network.de",
```
Comment (JakobBruening) [inline minecraft_servers/hardwarecraft/manifest.json:11]: Website is in maintenance - Please remove or update. Also it looks like a event announcement page, not like a support page.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/hardwarecraft/manifest.json (hunk 2 above)
Comment (NxrVincent) [reply minecraft_servers/hardwarecraft/manifest.json]: I removed the support website, is it all perfect now?

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- [ ] Invalid logo size! Size is 512x512px, shortest side must not exceed 256px: minecraft_servers/hardwarecraft/logo.png
- [ ] Error: Invalid background size! Size is 960x540px, must be 1280x720px: minecraft_servers/hardwarecraft/background.png
- [ ] Error: Invalid hDPI logo size! Size is 1024x1024px, shortest side must not exceed 512px: minecraft_servers/hardwarecraft/logo@2x.png

**Comment by NxrVincent** [comment]
We have updated the files, is everything correct now?

**Review by JakobBruening — APPROVED** [review]
Looks good - Thank you!

**Pull request merged**